awxkit/awxkit/api/pages/api.py

```diff
@@ -355,7 +367,10 @@ def import_assets(self, data):
         self._assign_membership()
         self._assign_roles()
 
-        return changed
+        return changed, self.get_error()
```

Comment:
Other code directly makes use of this method (see awx_collection/plugins/modules/import.py).  Changing the output is going to mess with those other callers, some of which we can't anticipate.  I haven't 100% thought this through yet, but I suspect what we actually want to do is to raise a `SystemExit` or do a `sys.exit` call right above here.  YMMV.

Reply:
Ditto with other places you changed the return signature to include the error bool, most likely.

Reply:
this seems to be safe and is the equivalent of doing

r, _ = command.handle(client, parser)

Reply:
I'm saying this is
- unnecessary
- and will involve **all** consuming code having to be adjusted from getting a single returned object to needing to destructure a tuple

Please don't do this.  Just raise a `SystemExit` inside this method and the `export_assets` method.

Reply:
To be clearer, the awx collection directly uses the `import_assets` and `export_assets` methods, without going through the `CustomCommand` interfaces provided for awx-cli.  We do not want to break the contract for these methods for applications and other consumers that make use of awxkit _as a library_, instead of invoking the CLI wrapper.